test/unit/plugins/validate-semantic/oas3/refs.js

```diff
@@ -206,4 +206,109 @@ describe("validation plugin - semantic - oas3 refs", () => {
       return expectNoErrorsOrWarnings(spec)
     })
   })
-})
+
+  describe("$refs for headers must reference a header by position", () => {
+    it("should return an error when a header incorrectly references a local parameter schema", () => {
```

Comment:
Naming suggestions for clarity:
```suggestion
  describe("response header $refs should not point to parameters", () => {
    it("should return an error when a response header incorrectly references a local parameter component", () => {
```

Reply:
Done!